packages/expo-cli/src/commands/build-native/index.ts

```diff
@@ -70,12 +73,136 @@ export async function buildAction(projectDir: string, options: Options): Promise
   }
 }
 
+async function logBuildStatuses(
+  username: string,
+  buildStatus: {
+    jobs: Array<Record<string, any>>;
+    canPurchasePriorityBuilds: boolean;
+    numberOfRemainingPriorityBuilds: number;
+    hasUnlimitedPriorityBuilds?: boolean;
+  }
+) {
+  log('=================');
+  log(' Builds Statuses ');
+  log('=================\n');
+
+  buildStatus.jobs.forEach((job, i) => {
+    let platform, packageExtension;
+    if (job.platform === 'ios') {
+      platform = 'iOS';
+      packageExtension = 'IPA';
+    } else {
+      platform = 'Android';
+      packageExtension = 'APK';
+    }
+
+    log(
+      `### ${i} | ${platform} | ${UrlUtils.constructBuildLogsUrl({
+        buildId: job.id,
+        username,
+      })} ###`
+    );
+
+    const hasPriorityBuilds =
+      buildStatus.numberOfRemainingPriorityBuilds > 0 || buildStatus.hasUnlimitedPriorityBuilds;
+    const shouldShowUpgradeInfo =
+      !hasPriorityBuilds &&
+      i === 0 &&
+      job.priority === 'normal' &&
+      buildStatus.canPurchasePriorityBuilds;
+    let status;
+    switch (job.status) {
+      case 'pending':
+      case 'sent-to-queue':
```

Comment:
Builds v2 don't have the same build statuses as the old ones.
Take a look at this file - https://github.com/expo/universe/blob/master/server/www/src/constants.ts#L20